packages/vm/tests/config.ts

```diff
@@ -347,8 +347,8 @@ const expectedTestsFull: any = {
     Byzantium: 4754,
     Constantinople: 10530,
     Petersburg: 10525,
-    Istanbul: 10759,
-    MuirGlacier: 10759,
+    Istanbul: 10715,
+    MuirGlacier: 10715,
```

Comment:
Any reason why the amount of tests here have decreased?

Reply:
TBH: these number checks are totally out of work anyhow, noticed this after the changes here. Istanbul actually has 11632 checks - see the respective [run](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1116/checks?check_run_id=1928421040). But the checks are not enforced and - furthermore - we also have this double structure with redundant test numbers being both in the `config.ts` and the CI workflow files in CI.

Don't know - I would have a tendency to fix this separately (or at least - to be plain honest - I am not motivated to do this here along this PR atm).